x-eithert/src/X/Control/Monad/Trans/Either.hs

```diff
@@ -0,0 +1,154 @@
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE RankNTypes #-}
+module X.Control.Monad.Trans.Either (
```

Comment:
Thing for people to keep in mind here, this will force the upgrade through all downstream projects when you upgrade